parquet-column/src/main/java/org/apache/parquet/column/values/bitpacking/ByteBitPackingValuesWriter.java

```diff
@@ -69,6 +70,11 @@ public void reset() {
   }
 
   @Override
+  public void close() {
+    /* nothing to do */
+  }
```

Comment:
Seems reasonable enough. It might be worth it to require the method be declared in a particular implementation if we anticipate subclasses frequently will have resources to close.